estate/models/estate_property.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "estate properties"
+    _order = "id"
+
+    name = fields.Char("Title", required=True)
+    description = fields.Text("Description")
+    postcode = fields.Char("Postcode")
+    date_availability = fields.Date("Available From")
+    expected_price = fields.Float("Expected Price", required=True)
+    selling_price = fields.Float("Selling Price")
+    bedrooms = fields.Integer("Bedrooms")
+    living_area = fields.Integer("Living Area ")
+    facades = fields.Integer("Facades")
+    garage = fields.Boolean("Garage")
+    garden = fields.Boolean("Garden")
+    garden_area = fields.Integer("Garden Area")
+    garden_orientation = fields.Selection(
+        string='Garden Orientation',
+        selection=[('north','North'),('south','South'),('east','East'),('west','West')]
```

Comment:
On the python-side, we try to follow the following for quoting strings:
- single quote for technical strings
- double-quote for human-displayed strings

=> `('north', "North")`

Let's try to follow it during the training - even though you'll find many counter examples in existing code.